doc/en/api/1.0.0/gwcbounds.yaml

```diff
@@ -0,0 +1,41 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Bounds
+  description: The REST API for Disk Quota management provides a RESTful interface through which get the bounds of a given layer based on srs.
```

Comment:
what does disk quota management have to do with this one?

Reply:
This was clumsy of me - fixed.